source/reporting/index.html.md.erb

```diff
@@ -69,7 +69,7 @@ When you select a payment, you can see the history of the payment. For example,
 You can also download a CSV spreadsheet of all your selected payments to help with [reconciliation](#check-your-payments-are-correct-39-reconciliation-39). You can download one report for all your services or one report for each separate service.
 
 
-You can also see payments made to your bank account by following the link from **My services** but only once you have [gone live](/switching_to_live) or set up a Stripe test account.
+You can also see payments made to your bank account by following the link from **My services**. This is only available for live Stripe accounts.
```

Comment:
Also for test Stripe accounts, sorry, which we said previously. Just thought it needed clarifying that it wasn't for *all* live accounts but only test Stripe accounts.